.github/workflows/rust.yml

```diff
@@ -25,6 +25,11 @@ jobs:
           - os: windows-latest
             features: '--features full'
         include:
+          # windows feature test
+          - os: windows-latest
+            toolchain: stable
+            profile: ''
+            features: '--features pcap-fopen-offline-precision'
```

Comment:
`pcap-fopen-offline-precision` won't work because WinPcap nor npcap export the function below:

```rust
// raw.rs
    #[cfg(feature = "pcap-fopen-offline-precision")]
    pub fn pcap_fopen_offline_with_tstamp_precision(arg1: *mut FILE, arg2: c_uint,
                                                    arg3: *mut c_char) -> *mut pcap_t;
```

Unfortunately it is not detected in current CI.